website/addons/wiki/templates/edit.mako

```diff
@@ -19,17 +19,17 @@
   <div class="row wiki-wrapper">
     <div class="panel-toggle col-sm-${'3' if 'menu' in panels_used else '1' | n}">
         <!-- Menu with toggle normal -->
-        <div class="wiki-panel wiki-panel-flex hidden-xs" style="${'' if 'menu' in panels_used else 'display: none' | n}">
-            <div class="wiki-panel-header wiki-panel-header-flex"> <i class="fa fa-list"> </i>  Menu
+        <div class="wiki-panel hidden-xs ${'' if 'menu' in panels_used else 'hidden-alt' | n}" data-bind="css: {  'wiki-panel-flex': $root.singleVis() === undefined, reset-height : $root.singleVis() !== undefined }">
+            <div class="wiki-panel-header" data-bind="css: {  'wiki-panel-header-flex': $root.singleVis() === undefined }"> <i class="fa fa-list"> </i>  Menu
```

Comment:
You can probably just check for the falsiness of `singleVis()` instead of comparing to `undefined`.